examples/again/src/lib.rs

```diff
@@ -384,12 +384,12 @@ impl IAudioProcessor for AGainProcessor {
             _ => kResultFalse,
         }
     }
-    unsafe fn get_latency_sample(&self) -> u32 {
+    unsafe fn get_latency_samples(&self) -> u32 {
         info!("Called: AGainProcessor::get_latency_sample()");
 
         0
     }
-    unsafe fn setup_processing(&self, setup: *mut ProcessSetup) -> tresult {
+    unsafe fn setup_processing(&self, setup: *const ProcessSetup) -> tresult {
```

Comment:
This is `ProcessSetup&` in the headers, should it be `const` here?

Reply:
Yes, as far as I have seen it's always read only. It's a simple struct with basic values like sample rate. The method setup_processing gets called always before processing in the state machine and it's a way for the host to inform the plugin before the processing step.